### ISSUE TITLE
Not Working, PLEASE FIX!!!!!

### ISSUE THREAD

**Issue by Pestermess** [victornpb/undiscord#615]
### Undiscord Version

5.2.3

### Browser

Firefox Latest

### Extension

Violent Monkey

### Reproduction steps

Clicked all the fields for deleting me, error pops up

### Actual results

Error searching messages, API responded with status 401!
	{"message":"401: Unauthorized","code":0}
CoreException	{}

### Expected results

Wiping the messages

### Bug affects

- [X] DMs
- [X] DMs (group)
- [X] Channels
- [X] Channel (NSFW)

### Additional information & file uploads

Error searching messages, API responded with status 401!
	{"message":"401: Unauthorized","code":0}
CoreException	{}

**Comment by PWDSerialgamer07** [comment]
Problem with discord API, they put some weird rate limit which breaks this.

**Comment by Pestermess** [comment]
> Problem with discord API, they put some weird rate limit which breaks this.

Makes sense. Hopefully the extension can be updated soon to work around Discord's API changes. I mainly use it to clean up messages from problematic users in our Discord servers.

**Comment by PesterMess** [comment]
Any update?

**Comment by dnemeth01** [comment]
https://github.com/victornpb/undiscord/discussions/589#discussioncomment-8376052
